### PR TITLE
exposeOpenApiAndPostmanCollection

### PR DESCRIPTION
**Pull request by AymericPlanche**
- update available events
- add postman getting started section
- remove the manually maintained api reference, make open api publicly available

### CODE REVIEW

**Comment by akeneo-secops** [comment]
### Gitleaks PR check
🟢 We detected no secrets within this PR! Good job 🎉

File: content/event-platform/api-reference.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# API reference
+
+## Open API specification
+
+[Download our Open API specification](https://storage.googleapis.com/akecld-prd-sdk-aep-dev-api-assets/openapi_specification.yml) to discover all the endpoints, and there request/response schema.
```
Comment (AymericPlanche) [inline content/event-platform/api-reference.md:5]: will be changed to production bucket when available.

File: content/event-platform/api-reference.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+You can use [the swagger online editor](https://editor-next.swagger.io/) to have a nice UI to render it.
```
Comment (AymericPlanche) [inline content/event-platform/api-reference.md:7]: will provide the static generated html file too in another pr

File: content/event-platform/getting-started.md [hunk 3]
```diff
@@ -1,18 +1,59 @@
...
+## Prerequisites
 
 Before proceeding, ensure that you have an active <a href="https://api.akeneo.com/apps/overview.html#whats-an-akeneo-app" target="_blank">App</a> (it can be a <a href="https://api.akeneo.com/apps/create-custom-app.html" target="_blank">Custom App</a> or an active <a href="https://api.akeneo.com/getting-started/connect-the-pim-4x/step-1.html#you-said-connection" target="_blank">connection</a>) to an Akeneo PIM.
 
 To learn how to create a connection, see the <a href="https://api-dev.akeneo.com/documentation/authentication.html#client-idsecret-generation" target="_blank">Authentication Guide</a>. If you're setting up a custom App, follow the steps in <a href="https://api-dev.akeneo.com/tutorials/how-to-get-your-app-token.html" target="_blank">this tutorial</a> to obtain an App token.
 
 ---
 
-### Basic Use Case: Setting Up Event Subscriptions
+## Using Postman
```
Comment (AymericPlanche) [inline content/event-platform/getting-started.md:13]: add a "using postman" at the same level than "using curl"

File: content/event-platform/getting-started.md [hunk 4]
```diff
@@ -1,18 +1,59 @@
...
 
-This guide provides a step-by-step walkthrough to help you set up event subscriptions using the Event Platform. By the end of this guide, you will have created a connection, authenticated your App, subscribed to events, and triggered an event from your PIM.
+The easiest way to manipulate the Event Platform API for a quickstart will be through our Postman Collection.
+
+### 1. Import the Postman Collection
+
+1. Download our [Postman Collection](https://storage.googleapis.com/akecld-prd-sdk-aep-dev-api-assets/generated_postman_collection.json)
```
Comment (AymericPlanche) [inline content/event-platform/getting-started.md:19]: links will be changed

File: content/event-platform/api-reference.md (hunk 1 above)
Comment (tomglvng) [inline content/event-platform/api-reference.md]:
```suggestion
[Download our Open API specification](https://storage.googleapis.com/akecld-prd-sdk-aep-dev-api-assets/openapi_specification.yml) to discover all the endpoints and their request/response schema.
```

Comment (clovis-akeneo) [inline content/event-platform/api-reference.md:5]:
```suggestion
[Download our Open API specification](https://storage.googleapis.com/akecld-prd-sdk-aep-dev-api-assets/openapi_specification.yml) to discover all the endpoints, and their request/response schema.
```

Comment (AymericPlanche) [reply content/event-platform/api-reference.md:5]: ALWAYS THIS ERROR, ALWAAAAYYYYSSSS

File: content/event-platform/getting-started.md [hunk 5]
```diff
@@ -1,18 +1,59 @@
 # Getting started
 
-This guide will walk you through the essential steps to set up your environment, create subscriptions, and trigger events.
+This guide provides a step-by-step walkthrough to help you set up event subscriptions using the Event Platform. By the end of this guide, you will have created a connection, authenticated your App, subscribed to events, and triggered an event from your PIM.
```
Comment (clovis-akeneo) [inline content/event-platform/getting-started.md:3]: :heart:

**Review by tomglvng — APPROVED** [review]

**Review by clovis-akeneo — APPROVED** [review]

**Pull request merged**